src/base/bittorrent/torrentinfo.cpp

```diff
@@ -213,24 +220,81 @@ QByteArray TorrentInfo::metadata() const
 
 QStringList TorrentInfo::filesForPiece(int pieceIndex) const
 {
-    if (pieceIndex < 0)
-        return QStringList();
+    // no checks here because fileIndicesForPiece() will return an empty list
+    QVector<int> fileIndices = fileIndicesForPiece(pieceIndex);
+
+    QStringList res;
+    res.reserve(fileIndices.size());
+    std::transform(fileIndices.begin(), fileIndices.end(), std::back_inserter(res), [this](int i) {
```

Comment:
Fix coding style for lambda. If you need split it you should use same style as for other functions:
```c++
[]()
{
    // body
}
```